yt_dlp/extractor/digitalconcerthall.py

```diff
@@ -0,0 +1,202 @@
+            + video_id, video_id, headers={'Accept': 'application/json',
+                                           'Accept-Language': language})
+        embedded = vid_info_dict.get('_embedded')
+        entries = []
+        for embed_type in embedded:
+            # embed_type should be either 'work' or 'interview'
+            # 'work' will be an array of one or more works
+            for item in embedded.get(embed_type):
+                if embed_type == 'interview':
+                    item['is_interview'] = 1
+                else:
+                    item['is_interview'] = 0
+                stream_href = item.get('_links').get('streams').get('href')
+                self.debug_out("JSON URL: " + 'https:' + stream_href)
+                test_dict = self._download_json('https:' + stream_href, video_id,
+                    headers={'Accept': 'application/json',
+                    'Authorization': 'Bearer ' + self._ACCCESS_TOKEN,
+                    'Accept-Language': language})
+                m3u8_url = test_dict['channel']['vod_mixed_hls_h264_hevc_abr_uhd']['stream'][0]['url']
+                self.debug_out('stream URL: ' + m3u8_url)
+
+                formats = self._extract_m3u8_formats(
+                    m3u8_url, video_id, 'mp4', 'm3u8_native', fatal=False)
+                self._sort_formats(formats)
+
+                if item.get('is_interview') == 1:
+                    title = "Interview - " + item.get('title', "unknown interview title")
+                else:
+                    title = (item.get('name_composer') if item.get('name_composer')
+                            else 'unknown composer') + ' - ' + item.get('title', "unknown title")
+                key = item.get('id')
+
+                duration = item.get('duration_total')
+                # avoid filenames that exceed filesystem limits
+                title = (title[:MAX_TITLE_LENGTH] + '..') if len(title) > MAX_TITLE_LENGTH else title
+                # append the duration in minutes to the title
+                title = title + " (" + str(round(duration / 60)) + " min.)"
+                self.debug_out("title: " + title)
+                timestamp = item.get('date').get('published')
+                entries.append({
+                    'id': key,
+                    'title': title,
+                    'url': m3u8_url,
+                    'formats': formats,
+                    'duration': duration,
+                    'timestamp': timestamp,
+                })
+                # use playlist description for video description by default
+                # but if the video has a description, use it
+                if test_dict.get('short_description'):
+                    entries[-1]['description'] = test_dict.get('short_description', "missing description")
+                if item.get('short_description'):
+                    entries[-1]['description'] = item.get('short_description', "missing description")
```

Comment:
absence of key must be represented as `None` value, not `"missing description"`
also, these two ifs can be merged

```suggestion
                description = test_dict.get('short_description') or item.get('short_description')
                if description:
                    entries[-1]['description'] = description
```